apps/dialtone-documentation/docs/components/popover.md

```diff
@@ -62,36 +62,106 @@ Your popover should be non-modal when:
   <example-popover modal />
 </code-well-header>
 
-```html
-<div class="d-modal--transparent"></div>
-<div class="d-popover">
-  <button id="anchor1" class="d-btn d-btn--primary" aria-expanded="false" aria-controls="dialog1" aria-haspopup="dialog">View Popover</button>
-  <div id="dialog1" class="d-popover__dialog d-popover__dialog--modal d-ps-relative d-t4 d-w264" role="dialog" aria-modal="true" aria-hidden="true" aria-labelledby="anchor1">
+<code-example-tabs
+htmlCode='
+<div class="d-popover d-fl-center d-fd-column d-p24 d-w100p d-of-auto">
+  <button id="anchor1" class="d-btn d-btn--primary" aria-expanded="true" aria-controls="dialog1" aria-haspopup="dialog">
+    View Popover
+  </button>
+  <div id="dialog1" class="d-popover__dialog d-popover__dialog--modal d-ps-relative d-t4 d-w264 d-hmx164 d-vi-hidden" role="dialog" aria-modal="true" aria-hidden="false" aria-labelledby="anchor1">
     <div class="d-popover__content d-p16">
-      This is content rendered within the popover.<br>
-      <button class="d-btn d-btn--primary">Button</button>
+      <div class="d-mb8">
+        This is content rendered within the popover.
+      </div>
+      <button class="d-btn d-btn--primary">
+        Button
+      </button>
     </div>
   </div>
 </div>
-```
+'
+vueCode='
+<dt-popover
+  :open="onOpen"
+  class="d-popover__dialog d-popover__dialog--modal d-ps-relative d-t4 d-w264 d-hmx164"
```

Comment:
Hmm why are we setting these classes on the vue example? They should be set via props

Ex 

```
:modal="true"
max-width="26.4rem"
```

Shouldn't have to set relative, d-t4 etc the popover vue component handles that itself.

This applies to the rest of the vue examples in this file as well.

Reply:
huh, right, seems like I copied it from `apps/dialtone-documentation/docs/.vuepress/exampleComponents/ExamplePopover.vue` .
Not necessary for the vue component, will remove it from the examples.